src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/BOMTable.tsx

```diff
@@ -57,14 +96,15 @@ const BOMTable: React.FC<BOMTableProps> = ({ columns, materials, assemblies }) =
     >
       <DataGrid
```

Comment:
can you change line 81 to be - 180px instead of 260px, I think we're scrunching up the table too much